### ISSUE TITLE
NumericFixedInterval ticks fail on inverted axes

### ISSUE THREAD

**Issue by Alexander-png** [ScottPlot/ScottPlot#3567]
**Issue:** when setting NumericFixedInterval for any axis and then inverting it the ticks and tick lines are missing.

**ScottPlot Version:** 5.0.23

**Here is the code for preproducing this issue:**
```cs
using ScottPlot.TickGenerators;
using System.Windows;

namespace ScottPlotBug
{
    public partial class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();

            Plotter.Plot.Axes.Left.TickGenerator = new NumericFixedInterval(10);
            Plotter.Plot.Axes.SetLimitsY(0, -100);

            Plotter.Plot.Axes.Bottom.TickGenerator = new NumericFixedInterval(10);
            Plotter.Plot.Axes.SetLimitsX(0, -100);
        }
    }
}
```
**Here what have been displayed:**
![image](https://github.com/ScottPlot/ScottPlot/assets/61787065/f58b4655-3721-4162-bc27-38f013c4e72b)

**Comment by swharden** [comment]
Hi @Alexander-png, I think the issue is that no ticks are drawn if there is no data on the plot. Add some data and ticks will appear:

```cs
using ScottPlot;

namespace Sandbox.WinForms;

public partial class Form1 : Form
{
    public Form1()
    {
        InitializeComponent();

        double[] xs = Generate.Consecutive(51);
        double[] ys = Generate.Sin(51, 10);
        formsPlot1.Plot.Add.Scatter(xs, ys);

        formsPlot1.Plot.Axes.SetLimitsY(10, -10);
        formsPlot1.Plot.Axes.SetLimitsX(50, 0);
    }
}
```

![image](https://github.com/ScottPlot/ScottPlot/assets/4165489/199232eb-fdf7-4325-8e78-4ad8e6d0d43f)

**Comment by swharden** [comment]
whoops, I realized the code example didn't properly test the tick generator. Let's try this again...

**Comment by swharden** [comment]
Fixed in 469ca0e66a3755d13e13b8ef921c02b4c9e4d263 - thanks again for pointing out this issue! The fix will be in the next release 🚀
